internal/testing/validation/pgadmin_test.go

```diff
@@ -32,6 +32,131 @@ func TestPGAdminInstrumentation(t *testing.T) {
 	assert.NilError(t, cc.Create(ctx, base.DeepCopy(), client.DryRunAll),
 		"expected this base to be valid")
 
+	t.Run("LogsBatches", func(t *testing.T) {
+		t.Run("Disable", func(t *testing.T) {
+			for _, tt := range []struct {
+				batches string
+				valid   bool
+			}{
+				{valid: true, batches: ``},              // both null
+				{valid: true, batches: `minRecords: 1`}, // one null
+				{valid: true, batches: `maxDelay: 1s`},  // other null
+
+				{valid: false, batches: `minRecords: 0`}, // one zero
+				{valid: false, batches: `maxDelay: 0m`},  // other zero
+
+				{valid: true, batches: `minRecords: 0, maxDelay: 0m`}, // both zero
```

Comment:
```suggestion
				{valid: true, batches: `minRecords: 0, maxDelay: 0m`}, // both zero
				{valid: true, batches: `minRecords: 1, maxDelay: 1s`}, // both non-zero
```
Just to complete the square/use test as documentation